tools/const_finder/lib/const_finder.dart

```diff
@@ -53,15 +51,22 @@ class _ConstVisitor extends RecursiveVisitor<void> {
 
   @override
   void visitInstanceConstantReference(InstanceConstant node) {
+    node.fieldValues.values.whereType<InstanceConstant>().forEach(visitInstanceConstantReference);
```

Comment:
This is important if the const is defined inline somewhere - it never gets visited by this method otherwise.

It has to be up here before we check for matches because it could either be in the class we care about or in some other class.